src/dcd/server.ts

```diff
@@ -46,7 +48,7 @@ export default class Server extends vsc.Disposable {
             additionsImports.push('-I' + item);
         })
 
-        if (process.platform === 'windows') {
+        if (process.platform === 'win32') {
             additionsImports.push('-IC:\\D\\dmd2\\src\\phobos');
```

Comment:
@mattiascibien I've added some configuration. On Linux and OS X the paths can vary... Thankfully, there is always a configuration file (generally `/etc/dmd.conf`) which contains the paths. On Windows however there is no configuration file, so import paths are directly configured. They are always the same though, from what I've seen.
So Windows and POSIX systems aren't exactly configured the same way, but everything should work out of the box.